dvc/scm/git/__init__.py

```diff
@@ -133,6 +134,7 @@ def ignore(self, path, in_curr_dir=False):
     @staticmethod
     def _add_entry_to_gitignore(entry, gitignore, ignore_list):
         content = entry
+        re.escape(content)
```

Comment:
@J0 are we sure that re.espace does the right thing? gitignore is not using regexps, it's a special format - https://git-scm.com/docs/gitignore#_pattern_format . Most likely we need to use some special tool or write our own escape function.